src/pages/TopicItem/TopicItem.js

```diff
@@ -1,21 +1,21 @@
 import * as React from 'react';
 
-import { MoreVert as MoreVertIcon } from '@mui/icons-material';
+import ArrowOutwardIcon from '@mui/icons-material/ArrowOutward';
```

Comment:
Let's group the imports
```suggestion
import {ArrowOutward as ArrowOutwardIcon} from '@mui/icons-material';
```